src/tests/ftest/container/attribute.py

```diff
@@ -103,18 +104,26 @@ def verify_get_attr(self, indata, outdata):
         """
         verify the Attributes value after get_attr
         """
+        decoded = {}
+        for key, val in outdata.items():
+            if isinstance(val, bytes):
+                # The API returns the values as bytes already.
+                decoded[key.decode()] = val
+            else:
+                # The JSON output encodes the bytes as base64, so
+                # we need to decode them for comparison.
+                decoded[key] = base64.b64decode(val)
```

Comment:
Does the `key` need to be decoded here as well?

Reply:
Nope. The attribute name is always stored and read as a string. It has to be decoded when it comes from the pydaos API, though.